tests/Query/SelectTest.php

```diff
@@ -254,6 +255,17 @@ public function testOrderByReset()
         $this->assertParams([], $select);
     }
 
+    public function testOrderByExpression()
+    {
+        $select = $this->factory
+            ->select()
+            ->from('users')
+            ->orderBy(express("FIELD(`status`, 'off')"), 'DESC');
```

Comment:
This is a bad example, as the `status` is quoted in a way that is specific to MySQL.